### PR TITLE
Project - Jorge Pascual Segovia

### PR DESCRIPTION
**Pull request by JorPS**
This is my first version of git project

### CODE REVIEW

File: _projects/2022/100486421/100486421.Rmd [hunk 1]
```diff
@@ -0,0 +1,146 @@
+---
+title: "Worldwide Top 10 listened songs in Spotify"
+description: |
+  A short description of the post.
```
Comment (Enchufa2) [inline _projects/2022/100486421/100486421.Rmd]: Please add a short description of the post.

**Review by Enchufa2 — CHANGES_REQUESTED** [review]
Thanks, now, I've reorganized your files, so first of all **click Pull** in RStudio's git panel to retrieve my commit before making further changes. The main file is now `_projects/2022/100486421/100486421.Rmd`.

Now, the idea is that this should be an article explaining the context, the process, and discussing the charts. See [your classmates' projects](https://csslab.uc3m.es/dataviz/projects.html) for some inspiration.

**Comment by JorPS** [comment]
I thought that was the objective of the presentations so the structure is very similar. I already comitted the changes but I don't know if you can see them or if I have to do some process again, Github is kind of a labrynth for me yet.

**Comment by Enchufa2** [comment]
Thanks! Merging after some cleanup.

**Comment by Enchufa2** [comment]
Published here: https://csslab.uc3m.es/dataviz/projects/2022/100486421/

**Pull request merged**